hbase-balancer/src/test/java/org/apache/hadoop/hbase/master/balancer/TestStochasticLoadBalancerBalanceCluster.java

```diff
@@ -51,6 +51,7 @@ public class TestStochasticLoadBalancerBalanceCluster extends StochasticBalancer
    */
   @Test
   public void testBalanceCluster() throws Exception {
+    conf.setLong("hbase.master.balancer.stochastic.maxRunningTime", 10_000);
```

Comment:
I made a lot of changes like this, wherever I could, to speed up our balancer test suite. We made a lot of assumptions that the balancer should run for 30s here, 60s here, and while the round numbers or less configuration is nice when writing tests, that runtime really adds up when running the whole suite locally